packages/python/plotly/pyproject.toml

```diff
@@ -1,3 +1,48 @@
 [build-system]
 requires = ["jupyterlab~=3.0;python_version>='3.6'", "setuptools>=40.8.0", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[project.urls]
+"HomePage" = "https://plotly.com/python/"
+"Documentation" = "https://plotly.com/python/"
+"Github" = "https://github.com/plotly/plotly.py"
+"Changelog" = "https://github.com/plotly/plotly.py/blob/master/CHANGELOG.md"
+
+[project]
+name = "plotly"
+authors = [
+    { name = "Chris P", email = "chris@plot.ly" }
+]
+maintainers = [
+    { name="Martha Cryan", email="martha.cryan@plot.ly" },
+    { name="Emily Kellison-Linn", email="emily@plot.ly" }
+]
+description = "An open-source interactive data visualization library for Python"
+readme = {file = "README.txt", content-type = "text/markdown"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: Visualization",
+    "License :: OSI Approved :: MIT License"
+]
+requires-python = ">=3.8"
+license = {text = "MIT License"}
+version = "6.0.0"
+dynamic = ["dependencies"]
+
+# TODO: check that this works
```

Comment:
Check this~!!

Reply:
Check the tar file to see what's included

Reply:
https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#custom-discovery